New version of terraform-docs (v0.12.0) Fails to Install in GitHub Actions

# 🐛 Summary #

I noticed an error during installation of [terraform-docs](https://github.com/terraform-docs/terraform-docs) in a [cisagov/action-apb](https://github.com/cisagov/action-apb) build dispatched to [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core/runs/2169568700?check_suite_focus=true#step:11:51).

## To reproduce ##

Steps to reproduce the behavior:

1. Run a workflow using `terraform-docs`
1. Watch it fail!

## Expected behavior ##

No issues running the workflow.

## Any helpful log output or screenshots ##

Error in the GHA log:

```log
Error: ../../../go/pkg/mod/github.com/terraform-docs/terraform-docs@v0.12.0/internal/format/asciidoc_document.go:14:2: package embed is not in GOROOT (/opt/hostedtoolcache/go/1.15.10/x64/src/embed)
```

Add any screenshots of the problem here.

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -25,7 +25,7 @@ jobs:
           python-version: 3.9
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.15'
+          go-version: '1.16'
       - name: Store installed Go version
         run: |
           echo "GO_VERSION="\

